src/olympia/addons/tests/test_models.py

```diff
@@ -338,7 +334,7 @@ def test_filter_for_many_to_many(self):
         assert collection.addons.count() == 0
 
         # Only deleted.
-        self.addon.update(is_listed=True)
+        self.make_addon_unlisted(self.addon, listed=True)
```

Comment:
I think there are a few tests that do something like `self.make_addon_unlisted(self.addon, listed=listed_variable)` though, which would mean some if ... logic.  Worth it for clarity?